modules/templates/proc-adding-templates.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * assemblies/assembly-admin-templates.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-adding-templates_{context}"]
+= Importing an existing template to {product}
+
+You can add an existing template to your {product} instance by using the Catalog Processor.
+
+.Prerequisites
+
+* You have created a directory or repository that contains at least one template YAML file.
+
+.Procedure
+
+. In the `app-config.yaml` configuration file, add the `catalog.configuration` section:
++
+[source,yaml]
+----
+# ...
+catalog:
+  locations:
+    - type: url # <1>
+      target: https://<repository_url>/templates # <2>
+    - type: url
+      target: https://<repository_url>/example-template.yaml
+   - type: file # <3>
+      target: ./templates # <4>
+   - type: file
+      target: ./templates/example-template.yaml
+# ...
+----
+<1> If you are importing templates from a repository, such as GitHub or GitLab, use the `url` type.
+<2> Specify the URL for either a directory where multiple templates are stored, or an individual template as a YAML file.
```

Comment:
Is this a correct statement?
`either a directory where multiple templates are stored`

I didn't know this was possible.  The way I've seen it done is a file that points to other files like it's done here:
https://github.com/janus-idp/software-templates/blob/main/showcase-templates.yaml

Reply:
Pretty sure I saw docs mentioning this which is why I included it, but I will leave it as individual files for now since I can't find the source atm.

Reply:
This is correct. You can import a template or a folder containing templates BUT then referenced in a Location YAML file
```
apiVersion: backstage.io/v1alpha1
kind: Location
metadata:
  name: templates
  description: Templates
spec:
  targets:
    - ./quarkus-application/template.yaml
    - ./quarkus-chatbot/template.yaml
    - ./quarkus-rest-client/template.yaml
    - ./quarkus-quickstart/template.yaml
```

Reply:
Right so the missing piece is the `Location` file

Reply:
I think that this expands the scope of the original issue enough to require its own Jira as an additional use case.